src/App.tsx

```diff
@@ -1,7 +1,39 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+interface AppState {
+  pressedKey: string | null;
+}
+
+export class App extends Component<{}, AppState> {
+  state: Readonly<AppState> = {
```

Comment:
TS will infer it from initial object
```suggestion
  state = {
```